open_humans/settings.py

```diff
@@ -207,6 +207,7 @@ def __str__(self):
     'allauth',
     'allauth.account',
     'allauth.socialaccount',
+    'allauth.socialaccount.providers.google',
```

Comment:
This looks like an accidental inclusion from the social login work. For clarity, I think it shouldn't be included here. :)

Reply:
oh, woops

Reply:
Fixed